spec/requests/potepan/products_spec.rb

```diff
@@ -4,19 +4,26 @@
 RSpec.describe 'Potepan::Products', type: :request do
   describe 'GET #show' do
     let(:product) { create(:product) }
+    let(:taxon) { create(:taxon) }
+    let(:related_products_list) { create_list(:product, 4, taxons: [taxon]) }
+    let(:related_products_addition) { create(:product, taxons:[taxon], price: "20.20") }
+    let(:related_products) { [related_products_list,related_products_addition] }
```

Comment:
この時点で`flatten`してあげればよいのではないでしょうか。
```suggestion
    let(:related_products) { [related_products_list,related_products_addition].flatten }
```

Reply:
- [ ] fixed